src/styles/colors.js

```diff
@@ -1,21 +1,31 @@
 export default {
-    white: '#FFFFFF',
-    gray1: '#FAFAFA',
-    gray2: '#ECECEC',
-    gray3: '#C6C9CA',
-    gray4: '#7D8B8F',
     dark: '#0b1b34',
     black: '#000000',
-    blue: '#0185ff',
-    blueHover: '#0063bf',
+    blue: '#5AB0FF',
+    blueHover: '#B0D9FF',
     floralwhite: '#fffaf0',
-    green: '#03d47c',
-    greenHover: '#03c775',
-    orange: '#ff7101',
-    pink: '#f68dfe',
-    red: '#fc3826',
-    redHover: '#e13826',
-    redDisabled: '#fea29a',
-    yellow: '#fed607',
+    orange: '#FF7101',
+    pink: '#F68DFE',
+    green: '#03D47C',
+    greenHover: '#00C271',
+    greenPressed: '#35DD96',
+    greenDisabled: '80E9BD',
```

Comment:
Should we name the colors just based off of the color (not the use for the color in the UI)?